feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/moods/attachPhotoActivity.java

```diff
@@ -0,0 +1,90 @@
+package com.example.feelslikemonday.ui.moods;
+
+import android.content.Context;
+import android.content.Intent;
+import android.database.Cursor;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.feelslikemonday.R;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class attachPhotoActivity extends AppCompatActivity {
+
+    public static final int CAMERA_REQUEST = 9999;
```

Comment:
You might want to use a string instead of an int. Nitpick, shouldn't be a concern

Reply:
the result i use integer, because when you start a new activity through intent, it requires to pass int instead of String.